src/Orleans.CodeGenerator/Model/SerializableTypeDescription.cs

```diff
@@ -173,7 +173,7 @@ public bool IsEmptyConstructable
                     t = t.BaseType;
                 }
 
-                foreach (var ctor in Type.Constructors)
+                foreach (var ctor in Type.Constructors.Where(x => x.IsImplicitlyDeclared))
```

Comment:
Will this break wire-compatibility for existing code?

Reply:
It seems to change the meaning of the property from any type which is has no ctor or ctor with no params, to no ctor or an imlicitly declared ctor with no params - so explicitly declared ctors with no params would be excluded

Reply:
Yep, that seems to have been a leftover from my many prior attempts at it :/.